package.json

```diff
@@ -39,6 +39,7 @@
     "i18next-xhr-backend": "^1.5.1",
     "lru_map": "^0.3.3",
     "marked": "^0.5.2",
+    "natives": "^1.1.6",
```

Comment:
@mikeller It fixes the stacktrace in the description (https://github.com/gulpjs/gulp/issues/2246)

Reply:
The schedule of  Node is a little strange. They release a version each 6 months (april and october) and the older, if covered by the LTS, then it has18 months of LTS and then 12 months of maintenance.
Seeing the graph it seems version 11 will be a short version, and will end soon:
![schedule](https://user-images.githubusercontent.com/2673520/50816343-8ef7da80-1320-11e9-9a73-5e26f8e6be68.png)

Version 12 will be a long version, and released october of this year, so we can ignore version 11, but I'm not too sure version 12 will fix these problems. If it has the same problems, then we can go with versions 11 and 12 because the problems and solutions will be the same.

By other side, it seems to me is that the Node 11 has a bug, and the natives is simply installing some version in advance of the actual version included in our Node, that seems to fix the problem, so I hope version 12 maybe can fix it. Do you understand the same than me?

This seems only affect Mac. Is that right? Maybe then, can it be added into `platformDependentModules` to limit the effect?

Reply:
@McGiverGim: Having to add `natives` as a direct dependency to make things work  in node 11 is ugly at least, but I'd much rather consider this a bug, so the hope is that this will be fixed at some stage.

Reply:
I was able to reproduce https://github.com/gulpjs/gulp/issues/2246 in our repository with the last release of Carbon (8.x) as well, however this could be due to a polluted local npm_modules directory or NPM cache from originally trying it with 11.x.

I'll try repro again. Sorry.